pkg/asset/internal/templates.go

```diff
@@ -774,6 +795,18 @@ spec:
             memory: 20Mi
             cpu: 10m
       dnsPolicy: Default  # Don't use cluster DNS.
+      tolerations:
```

Comment:
I would prefer to put them on master nodes as DNS pod is virtually part of the control plane, it's almost always turned on by default.